app/Console/Command/ExcludeMalformedLabel.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Label\Commands\ExcludeLabel as ExcludeLabelCommand;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use Doctrine\DBAL\Connection;
+use PDO;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+final class ExcludeMalformedLabel extends AbstractCommand
+{
+    private const LABEL_REGEX = '/^[a-zA-Z\d_\-]{2,50}$/';
+
+    private const MAX_RESULTS = 1;
+    private Connection $connection;
+
+    public function configure(): void
+    {
+        $this->setName('label:exclude-malformed');
+        $this->setDescription('Excludes malformed labels based on a regex');
+    }
+
+    public function __construct(CommandBus $commandBus, Connection $connection)
+    {
+        parent::__construct($commandBus);
+        $this->connection = $connection;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $labelsCount = $this->getAllLabelsCount();
+
+        $helper = $this->getHelper('question');
+        $updateQuestion = new ConfirmationQuestion('Check ' . $labelsCount . ' label(s)? [y/N] ', false);
+        if (!$helper->ask($input, $output, $updateQuestion)) {
+            return 0;
+        }
+
+        $progressBar = new ProgressBar($output, $labelsCount);
+
+        $offset = 0;
+        do {
+            $labels = $this->getAllLabels($offset);
+
+            foreach ($labels as $label) {
+                $labelId = new Uuid($label['uuid_col']);
+                $labelName = $label['name'];
+                $excluded = (bool) $label['excluded'];
+
+                if (!$excluded && !preg_match(self::LABEL_REGEX, $labelName)) {
```

Comment:
Exclude the already excluded labels? So update the methods `getAllLabelsCount` and `getAllLabels` to not take into account excluded labels.